CEXT-6421: update checkout starter kit docs for the 4-app split (#578)
AdobeDocs/commerce-extensibility: src/pages/starter-kit/checkout/totals-collector-fees.md

plugin.magento.out_of_process_totals_collector.api.get_total_modifications.custom_fees
```

> This webhook fires during cart totals collection, after shipping costs and discounts are computed, so your app receives both the computed shipping cost and any shipping discount as part of the request payload. Your app returns a JSON Patch response with the fees to apply. Commerce converts each fee to the store currency, adds the amounts to the cart grand total, and persists them for display in the checkout, admin documents, and transactional emails.

Reviewer: ```suggestion
This webhook fires during cart totals collection, after shipping costs and discounts are computed. Your app receives both the computed shipping cost and any shipping discount as part of the request payload. Your app returns a JSON Patch response with the fees to apply. Commerce converts each fee to the store currency, adds the amounts to the cart grand total, and persists them for display in the checkout, admin documents, and transactional emails.
```

> The `totals-collector` app doesn't declare this webhook or ship an example action for it — add your own runtime action implementing the response format below, then register it as described next.

Reviewer: ```suggestion
The `totals-collector` app does not declare this webhook or ship an example action for it. Instead, you must add your own runtime action implementing the response format below, then register it.
```
